spec/models/repository_spec.rb

```diff
@@ -16,6 +16,61 @@
   it { is_expected.to have_db_column(:name).of_type(:string).with_options(null: false) }
   it { is_expected.to have_db_column(:external_url).of_type(:string).with_options(null: false) }
 
+  describe 'scopes' do
+    describe '.only_mirrored' do
+      it 'returns only mirrored repos' do
+        mirrored = create :repository, mirroring_enabled: true
```

Comment:
maybe move things like this to `let()` or `let!()` blocks
also I would use `()`-brackets while calling `create` method  https://github.com/bbatsov/ruby-style-guide#method-invocation-parens bit it's more the matter of taste :wink: 
both applies to similar code below